sw/airborne/boards/baro_board_ms5611_i2c.c

```diff
@@ -64,7 +65,15 @@
 PRINT_CONFIG_VAR(BB_MS5611_TYPE_MS5607)
 
 struct Ms5611_I2c bb_ms5611;
+Butterworth2LowPass lowpass_pressure;
```

Comment:
Is it reasonable to filter the raw values inside the sensor drivers ? For instance some filters might better handle unfiltered data rather than filtered (and slightly delayed) data. I think this should be the task of the INS to pre-filter or not the sensor data. Also we have more sensors than navigation filters.
An exception would be the case of a sensor with know "issue" like peaks where a median filter for instance would help to remove outliers.

Reply:
I did this here cause this baro is horrible, the noise is really bad. The default configuration is already on maximum oversampling ratio so I was not sure what else to do. I can replace the lp filter with a median filter but I think that this raw signal is not reliable for any use...